src/components/ConnectPin.js

```diff
@@ -24,9 +25,34 @@ export default class ConnectPin extends React.Component {
         source={require('../../assets/images/pin-green.png')}
         style={styles.pin}
       />
-    ) : (
+    ) : this.props.branch == 'Army' ? (
       <Image
-        source={require('../../assets/images/pin.png')}
+        source={require('../../assets/images/pin-darkgreen.png')}
+        style={styles.pin}
+      />
+    ) : this.props.branch == 'Navy' ? (
+      <Image
+        source={require('../../assets/images/pin-blue.png')}
+        style={styles.pin}
+      />
+    ) : this.props.branch == 'Marines' ? (
+      <Image
+        source={require('../../assets/images/pin-purple.png')}
+        style={styles.pin}
+      />
+    ) : this.props.branch == 'Air_Force' ? (
+      <Image
+        source={require('../../assets/images/pin-red.png')}
+        style={styles.pin}
+      />
+    ) : this.props.branch == 'Coast_Guard' ? (
+      <Image
+        source={require('../../assets/images/pin-cyan.png')}
+        style={styles.pin}
+      /> 
+    ) :  (
+      <Image
+        source={require('../../assets/images/pin-white.png')}
```

Comment:
If you think you may reuse these colors sometime again it may be useful to define them in a helper/constants file in an array and just use that to map to the color once so you also don’t have 5 image components

Reply:
wait actually @lbkchen how do we change the color of the image without using more images? I don't think theres an easy way to do this.